.circleci/config.yml

```diff
@@ -145,13 +145,7 @@ commands:
       - run:
           name: Build for platform (docker build)
           command: |
-            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> TEST=1 VERBOSE=1
-      - early_return_for_forked_pull_requests
-      - run:
-          name: Build for platform (docker publish)
-          command: |
-            docker login -u redisfab -p $DOCKER_REDISFAB_PWD
-            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> TEST=1 VERBOSE=1 PUBLISH=1 DOCKERWRAPPER_EXTRA_VARS="DOCKER_PUSH_ONLY=1"
+            make -C build/docker build DOCKER_SUFFIX=".<<parameters.platform>>" OSNICK=<<parameters.platform>> ARTIFACTS=0
```

Comment:
Should be:
```
make build OSNICK=<<parameters.platform>> VERSION=$CIRCLE_TAG BRANCH=$CIRCLE_BRANCH TEST=1 OFFICIAL=1 SHOW=1
```

Reply:
Not yet - tests and things don't work. As mentioned - this restores the working behaviour - the next PR will add things. Let's not add scope to this PR.

Reply:
VERSION, BRANCH, OFFICIAL, SHOW are fine.